src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -373,6 +373,7 @@ private IEnumerable<AssociativeNode> GetDeltaAstListDeleted(IEnumerable<Subtree>
         private IEnumerable<AssociativeNode> GetDeltaAstListAdded(IEnumerable<Subtree> addedSubTrees)
         {
             var deltaAstList = new List<AssociativeNode>();
+            currentSubTreeList = new Dictionary<Guid, Subtree>();
```

Comment:
This causes some LiveRunner test cases fail. You may take to @junmendoza to confirm about this change...